src/frontend/screens/ExportObservations.tsx

```diff
@@ -0,0 +1,210 @@
+import {StyleSheet, TouchableOpacity, View} from 'react-native';
+import {BottomSheetWrapper} from '../sharedComponents/BottomSheetWrapper';
+import {HeaderText} from '../sharedComponents/Text/HeaderText';
+import {defineMessages, useIntl} from 'react-intl';
+import {BodyText} from '../sharedComponents/Text/BodyText';
+import {PrimaryButton, SecondaryButton} from '../sharedComponents/Buttons';
+import {NativeRootNavigationProps} from '../sharedTypes/navigation';
+import {DARK_GREY, LIGHT_GREY, WARNING_RED} from '../lib/styles';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import {useState} from 'react';
+import {useActiveProject} from '../contexts/ActiveProjectContext';
+import {UIActivityIndicator} from 'react-native-indicators';
+import {useObservations} from '../hooks/server/observations';
+import {useTracks} from '../hooks/server/track';
+import * as Sentry from '@sentry/react-native';
+import {useExportObservationsAndShare} from '../hooks/server/projects';
+
+const m = defineMessages({
+  close: {
+    id: 'screens.ExportObservations.close',
+    defaultMessage: 'Close',
+  },
+  download: {
+    id: 'screens.ExportObservations.download',
+    defaultMessage: 'Download',
+  },
+  allObservations: {
+    id: 'screens.ExportObservations.allObservations',
+    defaultMessage: 'All Observations',
+  },
+  allObservationsDescription: {
+    id: 'screens.ExportObservations.allObservationsDescription',
+    defaultMessage: 'Text only as a GeoJson file',
+  },
+  allObservationsAndMedia: {
+    id: 'screens.ExportObservations.allObservationsAndMedia',
+    defaultMessage: 'All Observations with Media',
+  },
+  allObservationsAndMediaDescription: {
+    id: 'screens.ExportObservations.allObservationsAndMediaDescription',
+    defaultMessage: 'Images and audio in a .zip file',
+  },
+  tracks: {
+    id: 'screens.ExportObservations.tracks',
+    defaultMessage: 'Tracks',
+  },
+  tracksDescription: {
+    id: 'screens.ExportObservations.tracksDescription',
+    defaultMessage: 'Tracks only as a GeoJson file',
```

Comment:
same nit about this being `GeoJSON` 😄 
```suggestion
    defaultMessage: 'Tracks only as a GeoJSON file',
```